docs/reference/known_limitations.md

```diff
@@ -4,13 +4,23 @@
 This section describes the known limitations associated with Percona Everest:
 
 
-
 ## Passwords
 
 Refrain from changing the password of administrative users (e.g., root, monitor, or operator) manually in the database. This action may cause inconsistencies with the secrets stored in Kubernetes, which are crucial for the proper functioning of the cluster. Such modifications have the potential to disrupt your cluster.
 
 We are developing a new feature that will allow you to modify these settings directly from the user interface (UI).
 
+## MongoDB sharding
+
+- Once MongoDB sharding is enabled, it cannot be disabled.
+
+
+## PSMDB operators
```

Comment:
PSMDB isn't a well-known acronym for users, I think we should refer to MongoDB instead.
```suggestion
## MongoDB
```